nltools/stats.py

```diff
@@ -26,7 +26,8 @@
             'make_cosine_basis',
             '_hc0',
             '_hc3',
-            '_hac']
+            '_hac',
```

Comment:
can you name these something more descriptive?

Maybe something like _robust_estimator_hac?

Reply:
Done